py-polars/polars/dataframe/frame.py

```diff
@@ -1602,12 +1602,12 @@ def raise_on_copy(msg: str) -> None:
 
         out = self._df.to_numpy(order)
         if out is None:
-            return np.vstack(
+            return np.column_stack(
```

Comment:
I can recall I benchmarked this one. Why do we change from `vstack` to `column_stack` and what are the perf implications?

Reply:
Let me revert this one for now and look into it more closely - I haven't looked into this too deeply but there are some correctness implications, e.g. currently Array types are not handled correctly.